my_project/my_app/views.py

```diff
@@ -1,23 +1,86 @@
 """
 Create your views here.
 """
+import json
+from hotels import find_express_deal
 from rest_framework.views import APIView
 from rest_framework.response import Response
 from rest_framework import status  # pylint: disable=unused-import
 
 
-class Sum(APIView):
+class BusinessLogic(APIView):
     """
-    Sample view class.
+    This is an example of adding/modifying slots using business logic, using
+    the `hotel_booking` competency. There is also an example at the very
+    beginning for arbitrary state transitions.
     """
     def post(self, request, _format=None):  # pylint: disable=unused-argument, no-self-use
         """
         Sample post request function.
-
-        Accepts a post request with two integers
-        and returns a reponse containing their sum.
         """
-        int1, int2 = (int(value) for value in request.data.values())
-        result = int1 + int2
-        return Response(result)
+        #overwrite state for state transition example
+        if request.data['state'] == "clean_hello":
+            request.data['state'] = "clean_goodbye"
```

Comment:
Could you design a slightly better example for this? Instead of just override the state if the current state is clean_hello, have a condition related to slots and states combination.

Reply:
I changed this to something more arbitrary and made up.  I think we should maybe remove 'hotels.py' if we are planning on giving them something they can't immediately test in Spotlight